bbc/simorgh
# WSTEAM1-659: Add iFrame component

Resolves JIRA [WSTEAM1-659](https://jira.dev.bbc.co.uk/browse/WSTEAM1-659)

Overall changes
======
Adds an iframe component and straps an 'iframe resize script' to the component so that the iframe automatically adjusts itself when its DOM is loaded.

![Screenshot 2023-10-25 at 11 19 31](https://github.com/bbc/simorgh/assets/32307964/d5a5e17b-9188-474f-9645-a1e3aba993f2)


Code changes
======
- _Adds `components/Embeds/EmbedIFrame/EmbedIFrame.tsx`, which populates all the necessary iframe attributes from the BFF response [here](https://github.com/bbc/fabl-modules/pull/5585)._

Testing
======
1. _Visit [https://www.test.bbc.com//pidgin/articles/c92r5nr8dd2o](https://www.test.bbc.com//pidgin/articles/c92r5nr8dd2o)_
2. _Make sure that the embed is shown at its full height._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
